### PR TITLE
update version-policy.md links

### PR DESCRIPTION
**Pull request by Arhell**
fix issue https://github.com/kubeflow/website/issues/2687
Removed broken links
after fix

**Review page**
https://deploy-preview-2693--competent-brattain-de2d6d.netlify.app/docs/reference/version-policy/

### CODE REVIEW

**Review by RFMVasconcelos — COMMENTED** [review]
Thank you for starting this @Arhell ! 

Please check my comments below :)

File: content/en/docs/reference/version-policy.md [hunk 1]
```diff
@@ -122,14 +122,14 @@ documentation for that application.
...
-        <td><a href="/docs/external-add-ons/serving/kfserving/">KFServing</a>
+        <td>KFServing
           (<a href="https://github.com/kubeflow/kfserving">GitHub</a>)
         </td>
         <td>Beta</td>
         <td>v0.3.0</td>
       </tr>
       <tr>
-        <td><a href="/docs/components/misc/metadata/">Metadata</a>
+        <td>Metadata
```
[the remark this answers is not in the document]
Comment (RFMVasconcelos) [reply content/en/docs/reference/version-policy.md:132]: I see https://github.com/kubeflow/website/issues/2631

File: content/en/docs/reference/version-policy.md [hunk 2]
```diff
@@ -100,7 +100,7 @@ documentation for that application.
         <td>1.0.0</td>
       </tr>      
       <tr>
-        <td><a href="/docs/components/training/chainer/">Chainer operator</a>
```
[the remark this answers is not in the document]
Comment (Arhell) [reply content/en/docs/reference/version-policy.md:103]: I think yes, deleted from the table

File: content/en/docs/reference/version-policy.md [hunk 3]
```diff
@@ -122,14 +122,14 @@ documentation for that application.
         <td>v1alpha3</td>
       </tr>
       <tr>
-        <td><a href="/docs/external-add-ons/serving/kfserving/">KFServing</a>
```
[the remark this answers is not in the document]
Comment (Arhell) [reply content/en/docs/reference/version-policy.md:125]: deleted from the table

File: content/en/docs/reference/version-policy.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (Arhell) [reply content/en/docs/reference/version-policy.md]: remove to

**Comment by Arhell** [comment]
@RFMVasconcelos all done, thank you
page
https://deploy-preview-2693--competent-brattain-de2d6d.netlify.app/docs/reference/version-policy/

File: content/en/docs/reference/version-policy.md [hunk 4]
```diff
@@ -121,20 +114,6 @@ documentation for that application.
         <td>Beta</td>
         <td>v1alpha3</td>
       </tr>
-      <tr>
-        <td><a href="/docs/external-add-ons/serving/kfserving/">KFServing</a>
-          (<a href="https://github.com/kubeflow/kfserving">GitHub</a>)
```
Comment (RFMVasconcelos) [inline content/en/docs/reference/version-policy.md:126]: @Arhell why delete kfserving?

File: content/en/docs/reference/version-policy.md [hunk 5]
```diff
@@ -239,7 +218,7 @@ one of the following Kubeflow SDKs and command-line interfaces
         <td>0.7.1</td>
       </tr>
       <tr>
-        <td><a href="/docs/methods/kfctl/kustomize/">kfctl</a> 
+        <td>kfctl
           (<a href="https://github.com/kubeflow/kfctl">GitHub</a> )
```
Comment (RFMVasconcelos) [inline content/en/docs/reference/version-policy.md:222]: Kfctl was also depricated in Kubeflow v1.3

File: content/en/docs/reference/version-policy.md (hunk 4 above)
Comment (Arhell) [reply content/en/docs/reference/version-policy.md:126]: returned and changed

File: content/en/docs/reference/version-policy.md (hunk 5 above)
Comment (Arhell) [reply content/en/docs/reference/version-policy.md]: removed

Comment (Arhell) [reply content/en/docs/reference/version-policy.md]: @RFMVasconcelos done

**Review by RFMVasconcelos — COMMENTED** [review]
Thank you for updating the PR @Arhell !! 

/lgtm
/approve

**Pull request merged**